generator/operation.go

```diff
@@ -398,9 +409,17 @@ func (b *codeGenOpBuilder) MakeOperation() (GenOperation, error) {
 		SuccessResponse: successResponse,
 		ExtraSchemas:    extra,
 		Schemes:         schemeOrDefault(schemes, b.DefaultScheme),
+		Produces:        producesOrDefault(produces, b.DefaultProduces),
 	}, nil
 }
 
+func producesOrDefault(produces []string, defaultConsumes string) []string {
```

Comment:
super tiny nitpick: I think `defaultConsumes` here wants to be called anything that doesn't have consumes in the name